src/ApiGenerator/RestSpecification/XPack/ccr.unfollow.json

```diff
@@ -1,7 +1,7 @@
 {
   "ccr.unfollow":{
     "documentation":{
-      "url":"https://www.elastic.co/guide/en/elasticsearch/reference/current/ccr-post-unfollow.html"
+      "url":"http://www.elastic.co/guide/en/elasticsearch/reference/current"
```

Comment:
Loss of information with url change?
https://www.elastic.co/guide/en/elasticsearch/reference/current/ccr-post-unfollow.html seems valid still...

Reply:
```suggestion
      "url":"https://www.elastic.co/guide/en/elasticsearch/reference/current/ccr-post-unfollow.html"
```